app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class InvitationsController < ApplicationController
+  before_action :require_user
+
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    @invitation = Invitation.new(invitation_params)
+    @invitation.author = current_user
```

Comment:
You could merge this line to previous as `@invitation = Invitation.new(invitation_params,merge!(author: current_user)`.